composer.json

```diff
@@ -11,7 +11,8 @@
     "require": {
         "php": ">=5.5",
         "symfony/swiftmailer-bundle": "~2.3",
-        "mailjet/mailjet-apiv3-php": "~1.1"
+        "mailjet/mailjet-apiv3-php": "~1.1",
+        "doctrine/orm": "^2.4.8"
```

Comment:
I have based my requirement based on the one in the adback project. I haven't looked to get a older version.

We could check an other one if you want